model/fv_arrays.F90

```diff
@@ -1489,14 +1532,45 @@ subroutine allocate_fv_atmos_type(Atm, isd_in, ied_in, jsd_in, jed_in, is_in, ie
     allocate (  Atm%ak(npz_2d+1) )
     allocate (  Atm%bk(npz_2d+1) )
 
+    allocate ( Atm%inline_mp%prew(is:ie,js:je) )
```

Comment:
We should consider only allocating arrays for the inline physics if the inline physics is active.

Reply:
@lharris4 is this a suggestion to be addressed now? Or would you like it addressed in a future release?

Reply:
Hi, Lauren. This is a recommendation for the next release. Thanks.

Reply:
I can do this in a future release.